### DISCUSSION TITLE
Zilliz Cloud只能通过oss批量导入么？

### DISCUSSION THREAD

**Discussion by LangKing9** [milvus-io/milvus#31762] [Q&A and General discussion]
直接在网页上没法批量导入对吧

**Comment by yhmo** [comment]
可以的
![Screenshot from 2024-04-01 10-20-51](https://github.com/milvus-io/milvus/assets/2282099/2fc3380b-871f-4473-8209-89bc1dd5f4a0)
![Screenshot from 2024-04-01 10-21-40](https://github.com/milvus-io/milvus/assets/2282099/659384a6-dc3e-40dc-b35a-c77e22ac8ba8)
